ush/config.sh.RRFS_NA_3km

```diff
@@ -112,6 +113,9 @@ for i in {3..23..6}; do FCST_LEN_HRS_CYCLES[$i]=24; done
 DA_CYCLE_INTERV="1"
 RESTART_INTERVAL="1 2"
 
+SST_ROOT="/lfs4/BMC/public/data/grids/ncep/sst/0p083deg/grib2"
+update_SST="01"
```

Comment:
Per my test, this can be a string. ksh will convert a string to an integer and then do the "eq" operation. I think the string form `update_SST="01"` reads better than `update_SST=01`